lib/src/Accordion/AccordionItem.tsx

```diff
@@ -18,31 +18,32 @@ export const AccordionItem: React.FC<AccordionItemProps> = React.memo(
 
         return (
             <div {...props} ref={ref} className={classnames("rc", "card", { collapsed: !props.defaultChecked }, props.className)}>
-                <div className="card-header" id={uniqueId + "-header"}>
+                <div className="card-header" id={`${uniqueId}--header`}>
```

Comment:
since we are referencing this line in line 40, can we set this to a const and reuse it for easier maintenance

Reply:
nice catch, refactored!